simularium/simularium-website
# Feature/add conversion form overlay

Problem
=======
Continuing work on autoconversion UI feature.

Closes #395 

Solution
========
with @meganrm 
* New feature (non-breaking change which adds functionality)


Change summary:
---------------
* ConversionForm component has been added
* Component has antd Select and Upload components for choosing file type and grabbing a local file
* Component renders through new import route
* Change in viewer state disables button in app header
* Light theme has been added to components folder and redundant styling was removed from modal specific css files
* Added Redux Logic to request template from simulariumio when simulation engine is selected
* Added selected engine template and selected file to state

Steps to Verify:
----------------
1. npm start
2. click Load Model button in header
3. Select 'import other file type'
4. Behold select and upload components

Keyfiles (delete if not relevant):
-----------------------
1. changes to a few files
2. to work with new import route changes were made to src/index.tsx , would appreciate extra scrutiny here to make sure i am not causing side effects
